Gemfile

```diff
@@ -3,21 +3,20 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 group :test do
-  gem 'coveralls',                 require: false
-  gem 'puppet_metadata', '~> 1.0', require: false
-  gem 'simplecov-console',         require: false
-  gem 'voxpupuli-test', '~> 5.4',  require: false
+  gem 'coveralls',                  require: false
+  gem 'puppet_metadata', '~> 4.0',  require: false
+  gem 'simplecov-console',          require: false
+  gem 'voxpupuli-test', '~> 9.0',   require: false
 
   gem 'puppet-lint-package_ensure-check', '~> 0.2.0',          require: false
-  gem 'puppet-lint-resource_reference_syntax', '~> 1.2.0',     require: false
-  gem 'puppet-lint-strict_indent-check', '~> 2.1.0',           require: false
-  gem 'puppet-lint-unquoted_string-check', '~> 2.2.0',         require: false
-  gem 'puppet-lint-variable_contains_upcase', '~> 1.4.0',      require: false
-
-  gem 'facterdb', '< 2', require: false # 3.0.0 drops support for EL7
-
-  gem 'r10k',    require: false
-  gem 'toml-rb', require: false # puppet/telegraf
+  gem 'puppet-lint-resource_reference_syntax', '~> 2.0.0',     require: false
```

Comment:
that's a dependency of voxpupuli-puppet-lint-plugins and that's a dependency of voxpupuli-test. So you can drop it I think?

Reply:
I swear that wasn't the case back when I originally added the lint gems. It looks like things have changed a bit:

```
    voxpupuli-puppet-lint-plugins (5.0.0)
      puppet-lint (~> 4.0)
      puppet-lint-absolute_classname-check (~> 4.0)
      puppet-lint-anchor-check (~> 2.0)
      puppet-lint-file_ensure-check (~> 2.0)
      puppet-lint-leading_zero-check (~> 2.0)
      puppet-lint-lookup_in_parameter-check (~> 2.0)
      puppet-lint-manifest_whitespace-check (~> 0.3, < 1.0.0)
      puppet-lint-optional_default-check (~> 2.0)
      puppet-lint-param-docs (~> 2.0)
      puppet-lint-param-types (~> 2.0)
      puppet-lint-params_empty_string-check (~> 2.0)
      puppet-lint-resource_reference_syntax (~> 2.0)
      puppet-lint-strict_indent-check (~> 3.0)
      puppet-lint-topscope-variable-check (~> 2.0)
      puppet-lint-trailing_comma-check (~> 2.0)
      puppet-lint-unquoted_string-check (~> 3.0)
      puppet-lint-variable_contains_upcase (~> 2.0)
      puppet-lint-version_comparison-check (~> 2.0)
```

I appears everything except `puppet-lint-package_ensure-check` can be removed.